app/controllers/admin/volunteers.py

```diff
@@ -92,13 +105,58 @@ def addVolunteer(volunteer, eventId):
         flash("Error when adding volunteer", "danger")
     return ""
 
+@admin_bp.route('/addOutsideParticipantToEvent/<outsideParticipant>/<eventId>', methods = ['POST'])
+def addOutsideParticipant(outsideParticipant, eventId):
```

Comment:
Please rename this to describe what the function is doing, because we are not really adding outside participants in this function.

Reply:
resolved

Reply:
not resolved